app/controllers/questions.rb

```diff
@@ -0,0 +1,20 @@
+get '/questions' do
+  # show all questions
+  # on the board we wrote this route as "/questions/index"
+  # erb:'questions/show'
```

Comment:
I think this is typically questions/index view & an individual question is the questions/show view